tests/installation/boot_into_snapshot.pm

```diff
@@ -23,10 +23,9 @@ use base "opensusebasetest";
 
 sub run {
     my ($self) = @_;
-    assert_screen 'linux-login', 200;
+
     select_console 'root-console';
     # 1)
-    assert_script_run('touch /etc/NOWRITE;test ! -f /etc/NOWRITE');
```

Comment:
This line cannot be removed.

`grub_test` is booting into a snapshot if var `BOOT_INTO_SNAPSHOT` is present.
There is only one writable snapshot, and all of the snapshots under the "Start bootloader from a read-only snapshot" are read-only. If this command fails, it is a bug in the product or in the test.

I assume this line was removed because it caused a failure in a verification run. Could you paste the link to it?

Reply:
this is a product issue?

please see http://openqa.suse.de/tests/4636830#step/boot_into_snapshot/9

Reply:
No, The expected behavior is that the file is NOT created, because the snapshot is a read-only filesystem.

https://doc.opensuse.org/documentation/leap/reference/html/book.opensuse.reference/cha-snapper.html#sec-snapper-snapshot-boot

Reply:
@Zaoliang, the link you provided passed. It tried to create a file `/etc/nowrite`, and if was catch by the `! -f...` part. The error message is `touch: cannot touch '/etc/NOWRITE': Read-only file system. Which is expected.

Why did you the remove the line?

Reply:
That was the reason:

http://10.162.23.47/tests/8107#step/boot_into_snapshot/11
Now I need to think about this why this happened.

Reply:
I think, after first_boot and opensuse_welcome the snapshot becomes writable.

Reply:
Please see: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10919#issuecomment-688232334